react-ui/src/pages/contest/Contest.jsx

```diff
@@ -1,227 +1,64 @@
 /**
  * Voting and winners
+ * ??? Contest fields as props vs useSwrData
+ * ??? apiPath as prop vs hooks and rederive
+ * ??? remove winners / top20?
+ * ??? Logic in useEffect
```

Comment:
First two comments were wondering whether it is more efficient to just pass the contest information as props (e.g. `<Contest {...{contestId, data}}>`.... but since I don't have a firm grasp on router/web-based navigation and state it probably best to keep as is. Even on mobile they generally say to use hooks to access state. I'll delete the comments

Second comment was wondering if the top 20 should be removed in favor of standardizating the display. Not a priority.

Third comment was to see if the useEffect logic could be simplified now that we switched to modal.

I am going to delete all 4 of these comments, but if you think anything should be changed please add them to the ToDos

Reply:
I think removing the top 20 should be done at some point, but it's not high priority. I think I have some of the detailed results for the older contests that I can eventually add to the database to make it consistent with the newer contests. Created #273.

The useEffect stuff for positioning? Might be worth taking a look at that too some time down the line. Created #274.